packages/contracts/contracts/universal/StandardBridge.sol

```diff
@@ -208,14 +223,45 @@ abstract contract StandardBridge {
         uint256 _amount,
         bytes calldata _data
     ) public onlyOtherBridge {
```

Comment:
Previously, not all revert cases were being caught. With the `try/catch`, it guarantees that all revert cases are caught. The downside is that you can only `try/catch` on public functions, so it needs to be restricted with an `onlySelf` modifier and have the additional check that `localToken` isn't calling itself. I think that this is overall more robust and will guarantee any reverting conditions do not result in users getting their tokens/ETH stuck